cvat/apps/documentation/user_guide.md

```diff
@@ -1024,6 +1025,54 @@ or shortcuts:
 In order to change the zoom level, go to settings (press ``F3``)
 in the workspace tab and set the value Attribute annotation mode (AAM) zoom margin in px.
 
+## AI Tools
+
+The tool is designed for semi-automatic and automatic annotation using DL models.
+The tool is available only if there is a corresponding model.
+For more details about DL models read the [Models](#models) section.
+
+### Interactors
+
+Interactors are used to create a polygon semi-automatically.
+Supported DL models are not bound to the label and can be used for any objects.
+To create a polygon you have to specify positive and / or negative points.
```

Comment:
Let's rephrase: "To create a polygon usually you need to use regular or  positive points. For some kinds of segmentation negative points are available"